app/controllers/OrderController.java

```diff
@@ -3,30 +3,39 @@
 import io.sphere.sdk.carts.Cart;
 import io.sphere.sdk.products.ProductVariant;
 import models.OrderPageData;
-import play.Configuration;
+import play.Application;
 import play.Logger;
 import play.mvc.Result;
 import services.CartService;
 import views.html.order;
 import views.html.success;
 
+import javax.inject.Inject;
+
 import static java.util.Objects.requireNonNull;
 
 public class OrderController extends BaseController {
 
+    private static final Logger.ALogger LOG = Logger.of(OrderController.class);
```

Comment:
Great, I think the default logger of play is not suitable for maintainable apps.